src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.awt.*;
+
+public abstract class Figure {
+    String color;
+
+    public abstract void draw();
```

Comment:
The `Figure` class should also define an abstract method for calculating the area, such as `public abstract double getArea();`, to ensure that all subclasses implement this functionality.